### PR TITLE
new Zero logo

### PR DESCRIPTION
**Pull request by caneco**
created a new logo inspired from Laravel logo family designed by @sschoger

… and added it to the README file

### CODE REVIEW

**Comment by introwit** [comment]
@caneco thank you for the contribution :) But the logo isn't visible on your fork.

**Comment by caneco** [comment]
@introwit dang… github don't allow base64 images. What could be the best approach to host the svg file?

BTW the logo looks like this:
logo-zero

**Comment by nunomaduro** [comment]
Wow! I must admit that I am in ❤️ with the logo!

**Comment by caneco** [comment]
do you get the subliminal message behind the shape? 😄

**Comment by introwit** [comment]
@nunomaduro would be the appropriate person to take the call on the logo. What I can suggest is move the logo credits below in the `Credits` section of the readme.

File: readme.md [hunk 1]
```diff
@@ -15,6 +14,8 @@ About Laravel Zero
 Laravel Zero was created by [Nuno Maduro](https://github.com/nunomaduro) and it is maintained by [Nuno Maduro](https://github.com/nunomaduro) & [Harish Toshniwal](https://github.com/introwit). Laravel Zero is a micro-framework that provides an elegant starting point for your next console application.
 **Unofficial** and customized version of Laravel optimized for building console/shell/command-line applications.
 
+Laravel Zero logo was made by [Caneco](https://github.com/caneco), inspired by the Laravel logo family designed by [@steveschoger](https://twitter.com/steveschoger).
```
Comment (nunomaduro) [inline readme.md]: You can remove this. I will create an CREDITS.MD to all maintainers, contributors, etc. Then I will add the credits for this logo.

File: readme.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
...MgMS42Yy0uOC0uMi0xLjYtLjMtMi40LS4zLTEuNiAwLTIuOC43LTMuOCAycy0xLjUgMy0xLjUgNXYxMC4yaC0xLjd2LTE4LjNoMS40bC4yIDMuM2guMWMuOC0xLjMgMS42LTIuMyAyLjQtMi44LjktLjYgMS45LS45IDMuMS0uOXpNMjcwIDMxLjhjMCAzLS43IDUuMy0yLjIgNy0xLjQgMS43LTMuNSAyLjUtNiAyLjUtMS42IDAtMy0uNC00LjMtMS4yLTEuMi0uOC0yLjItMS45LTIuOC0zLjMtLjctMS40LTEtMy4xLTEtNSAwLTMgLjctNS4zIDIuMi03IDEuNS0xLjcgMy40LTIuNSA2LTIuNSAyLjUgMCA0LjUuOCA1LjkgMi41IDEuNSAxLjggMi4yIDQuMSAyLjIgN3ptLTE0LjUgMGMwIDIuNS42IDQuNSAxLjcgNS45IDEuMSAxLjQgMi43IDIuMSA0LjcgMi4xczMuNi0uNyA0LjctMi4xYzEuMS0xLjQgMS43LTMuNCAxLjctNS45IDAtMi41LS42LTQuNS0xLjctNS45LTEuMS0xLjQtMi43LTIuMS00LjctMi4xcy0zLjYuNy00LjcgMi4xYy0xLjEgMS40LTEuNyAzLjQtMS43IDUuOXoiLz48bGluZWFyR3JhZGllbnQgaWQ9IlNWR0lEXzFfIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgeDE9IjI4IiB4Mj0iMjgiIHkyPSI1NiI+PHN0b3Agb2Zmc2V0PSIwIiBzdG9wLWNvbG9yPSIjNjhiZjY2Ii8+PHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjNDJiMjU3Ii8+PC9saW5lYXJHcmFkaWVudD48cGF0aCBmaWxsPSJ1cmwoI1NWR0lEXzFfKSIgZD0iTTQ4IDEzLjRsOC04di0xLjRjMC0yLjItMS44LTQtNC00aC00OGMtMi4yIDAtNCAxLjgtNCA0djM4LjZsOC04di0yNi42aDM0LjZsLTI2LjYgMjYuNi04IDgtOCA4djEuNGMwIDIuMiAxLjggNCA0IDRoNDhjMi4yIDAgNC0xLjggNC00di0zOC42bC04IDh2MjYuNmgtMzQuNmwxOS45LTE5LjkgMTQuNy0xNC43eiIvPjwvc3ZnPg=="></p>
```
Comment (nunomaduro) [inline readme.md]: Like we discuss on twitter, you will send this same logo on PNG format in different sizes. If you want, you may commit them to the docs repo and then add the src image of the png here.

Comment (caneco) [reply readme.md]: I'm to create the pack of versions of the logo… Sorry about the delay.

Comment (caneco) [reply readme.md]: Here you have…
https://www.dropbox.com/s/edqqbdt26qscwup/laravel-zero-set.zip?dl=0

About the PR… maybe its better to close the PR for you to organize both image in the `laravel-zero/docs` repo and the README file.

D'accord? 😃

**Comment by nunomaduro** [comment]
Thank you so much @caneco for this contribution! I will organise this tomorrow! ❤️

**Pull request closed without merging**